CONTRIBUTING.rst

```diff
@@ -619,6 +619,24 @@ learn how to use those tools to build documentation.
 
 .. _git_repo:
 
+
+Continuous integration
```

Comment:
It is getting hard to remember all the ways we can control the CI via commit message so adding a table.

Stole the idea from scikit learn:
https://scikit-learn.org/stable/developers/contributing.html#continuous-integration-ci

Reply:
This is great!